integration-cli/check_test.go

```diff
@@ -125,17 +74,17 @@ func TestMain(m *testing.M) {
 	for _, img := range images {
 		protectedImages[img] = struct{}{}
 	}
-	if !isLocalDaemon {
-		fmt.Println("INFO: Testing against a remote daemon")
-	} else {
+	if testEnv.LocalDaemon() {
```

Comment:
Similar to @dnephin's comments `testEnv.IsLocalDaemon()` may be clearer, but that's just a nit